src/components/TaskList/TaskList.jsx

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable react/prop-types */
+import React,{useState} from 'react'
+import styles from '@/components/TodoList/TodoList.module.css'
+
+function TaskList({keyId,value,editListHandler,removeListHandler}) {
+
+    const [editTaskInput,setEditTaskInput ] = useState(value)
+    const [checkBox,setCheckBox] = useState(false);
+    const [edit,setEdit] = useState(false);
+
+    const editTask = (event) => {
+        if(event.key == 'Enter' && checkBox != true){
+        editListHandler(event);
+        setEdit(false);
+        }
+    }
+
+    const completed = (event) => {
+        if(event.target.checked) setCheckBox(true) 
+        else setCheckBox(false)
```

Comment:
here instead of an if else you could use a ternary operator. And in this case as event.target.checked is a boolean you can directly set that.